### ISSUE TITLE
Add free-programming-books

### ISSUE THREAD

**Issue by JuanPabloDiaz** [marcelscruz/dev-resources#543]
I would like to add this great resource but I am not sure where will be a good place to add it? ...Books?

it's a **HUGE** List of Free Learning Resources In Many Languages

📚 Freely available programming books

https://ebookfoundation.github.io/free-programming-books/

**Comment by marcelscruz** [comment]
Yes @JuanPabloDiaz, I think "Book" and "Learn" are the appropriate categories for it.
Thanks for bringing this up!

**Comment by marcelscruz** [comment]
Closing this for now, feel free to add the resource whenever you're ready.
